go.mod

```diff
@@ -140,7 +140,7 @@ require (
 	github.com/google/go-querystring v1.1.0 // indirect
 	github.com/google/gofuzz v1.2.0 // indirect
 	github.com/google/s2a-go v0.1.8 // indirect
-	github.com/googleapis/enterprise-certificate-proxy v0.3.3 // indirect
+	github.com/googleapis/enterprise-certificate-proxy v0.3.4 // indirect
```

Comment:
Is this really needed for the info function?

Reply:
I upgraded it because Prometheus won't build on my MacBook otherwise (I've only got the MacBook currently):
```console
✗ go mod tidy
go: downloading github.com/googleapis/enterprise-certificate-proxy v0.3.3
go: github.com/prometheus/prometheus/storage/remote/googleiam imports
	google.golang.org/api/transport/http imports
	google.golang.org/api/internal/cert imports
	github.com/googleapis/enterprise-certificate-proxy/client: github.com/googleapis/enterprise-certificate-proxy@v0.3.3: verifying module: checksum mismatch
	downloaded: h1:G6q7VHBoU74wQHXFsZSLMPl0rFw0ZDrlZ3rt6/aTBII=
	sum.golang.org: h1:QRje2j5GZimBzlbhGA2V2QlGNgL8G6e+wGo/+/2bWI0=

SECURITY ERROR
This download does NOT match the one reported by the checksum server.
The bits may have been replaced on the origin server, or an attacker may
have intercepted the download attempt.
```

It's not a real upgrade, since nothing changes in [v0.3.4](https://github.com/googleapis/enterprise-certificate-proxy/releases/tag/v0.3.4), it just fixes that the v0.3.3 release was inadvertently changed, causing the above error:

> This is strictly a version bump release to fix github mirrors issue associated with v0.3.3 (due incorrectly tagged commit id). No code changes between v0.3.3 and v0.3.4.

Reply:
Separate PR please.

Reply:
OK, will make one.

Reply:
See [PR](https://github.com/prometheus/prometheus/pull/15166) please.